src/mailing/templates/forgot_password/html.ejs

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Password Reset</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
```

Comment:
`email-templates` library will be looking for the file starting from the `templates` directory, so we have to write this path/href relative to that folder:

```suggestion
    <link rel="stylesheet" type="text/css" href="forgot_password/style.css" />
```